src/Tribe/Views/V2/Hooks.php

```diff
@@ -87,6 +87,7 @@ protected function add_filters() {
 		add_filter( 'tribe_general_settings_tab_fields', [ $this, 'filter_general_settings_tab_live_update' ], 20 );
 		add_filter( 'tribe_events_rewrite_i18n_slugs_raw', [ $this, 'filter_rewrite_i18n_slugs_raw' ], 50, 2 );
 		add_filter( 'tribe_get_event_after', [ $this, 'filter_events_properties' ] );
+		add_filter( 'tribe_template_file', [ $this, 'filter_template_file' ], 10, 3 );
 
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: 13cac57bb2d806efa1e71d4959d92706</sub>